src/services/importedJsonTypes.ts

```diff
@@ -64,10 +70,14 @@ export interface TopLevelLicenseDetection {
   detection_count: number;
   detection_log?: string[] | null;
   file_regions?: LicenseFileRegion[];
-  matches?: LicenseDetectionMatch[];
+  reference_matches?: LicenseDetectionMatch[];
+
+  // Parser-added fields
+  matches?: LicenseDetectionMatch[]; // Also part of legacy output
```

Comment:
these ensure correct importing of legacy json files, for example in toolkit v32.0.8-51-g84d96507f8 we had `sample_matches` instead of `reference_matches`